src/webgpu/api/validation/createBindGroupLayout.spec.ts

```diff
@@ -64,6 +64,33 @@ g.test('visibility_and_dynamic_offsets')
     }, !success);
   });
 
+g.test('mininum_buffer_binding_size')
+  .params(
+    params()
+      .combine(poptions('type', kBindingTypes))
+      .combine(poptions('minBufferBindingSize', [0, 4]))
```

Comment:
I think we should test `undefined` here in order to test the default value of this option.
```suggestion
      .combine(poptions('minBufferBindingSize', [undefined, 0, 4]))
```

Most of the time I think these will go in operation tests, but in this case it only really affects validation.